solr/core/src/java/org/apache/solr/cli/PackageTool.java

```diff
@@ -342,6 +341,14 @@ public List<Option> getOptions() {
             .required(false)
             .desc("Don't prompt for input; accept all default choices, defaults to false.")
             .longOpt("noprompt")
+            .build(),
+        // u was taken, can we change that instead?
+        Option.builder("credentials")
```

Comment:
same as above, why not `SolrCLI.OPTION_CREDENTIALS`?

Reply:
likewise, the `u` already used...